implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -384,14 +412,24 @@ public String getMpJwtIssuer() {
         return mpJwtRequireIss;
     }
 
-    public String getTokenHeader() {
+    public Optional<String> getTokenHeader() {
         return tokenHeader;
     }
 
     public Optional<String> getTokenCookie() {
         return tokenCookie;
     }
 
+    @Deprecated
```

Comment:
Lets drop these two methods, we don't really need them

Reply:
Ok. I think we are not using the other getters as well. So maybe one more reason to refactor this :)

Reply:
They are for Thorntail, some of them, so I'm just keep adding getters in case it ever needs them

Reply:
Ok.